libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/LanguageGenerationBotComponent.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Microsoft.Bot.Builder.Dialogs.Adaptive.Templates;
+using Microsoft.Bot.Builder.Dialogs.Declarative;
+using Microsoft.Bot.Builder.Dialogs.Declarative.Converters;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Bot.Builder.Dialogs.Adaptive
+{
+    /// <summary>
+    /// ComponentRegistration class for language generation resources.
+    /// </summary>
+    public class LanguageGenerationBotComponent : BotComponent
```

Comment:
I wonder whether the naming is wrong???

In the underlying framework we have:

```
class Startup
{
  public void ConfigureServices(IServiceCollection services)
```
and now you have function StartUp